pkg/types/metricsmoduletypes/metricsmoduletypes.go

```diff
@@ -0,0 +1,73 @@
+package metricsmoduletypes
+
+// Order specifies column ordering preferences for stats queries.
+type OrderBy struct {
+	ColumnName string `json:"columnName"`
+	Order      string `json:"order"`
+}
+
+// StatsRequest represents the payload accepted by the metrics stats endpoint.
+type StatsRequest struct {
+	Expression string   `json:"expression"`
+	Start      int64    `json:"start"`
+	End        int64    `json:"end"`
+	Limit      int      `json:"limit"`
+	Offset     int      `json:"offset"`
+	OrderBy    *OrderBy `json:"orderBy,omitempty"`
+}
+
+// MetricStat represents the summary information returned per metric.
+type MetricStat struct {
+	MetricName   string `json:"metric_name"`
+	Description  string `json:"description"`
+	MetricType   string `json:"type"`
+	MetricUnit   string `json:"unit"`
+	TimeSeries   uint64 `json:"timeseries"`
+	Samples      uint64 `json:"samples"`
+	LastReceived int64  `json:"lastReceived"`
+}
+
+// StatsResponse represents the aggregated metrics statistics.
+type StatsResponse struct {
+	Metrics []MetricStat `json:"metrics"`
+	Total   uint64       `json:"total"`
+}
+
+type MetricMetadata struct {
+	Description string `json:"description"`
+	MetricType  string `json:"type"`
+	MetricUnit  string `json:"unit"`
+	Temporality string `json:"temporality"`
+}
```

Comment:
Where is monotonous property of metric in the response? Are we missing any other metadata?